examples/k8s-cronjob.md

```diff
@@ -20,52 +35,48 @@ kind: CronJob
-  failedJobsHistoryLimit: 5
-  successfulJobsHistoryLimit: 5
- concurrencyPolicy: Forbid
+  schedule: "*/15 * * * *"  # Run every 15 minutes
+  failedJobsHistoryLimit: 5  # Keep history of failed jobs
+  successfulJobsHistoryLimit: 5  # Keep history of successful jobs
+  concurrencyPolicy: Forbid  # Do not allow concurrent jobs
   jobTemplate:
     spec:
-        template:
-            spec:
-                restartPolicy: Never
-                containers:
-                - name: fedifetcher
-                  image: ghcr.io/nanos/fedifetcher:latest
-                  imagePullPolicy: IfNotPresent
-                  env:
-                    - name: FF_HOME_TIMELINE_LENGTH
-                      value: "200"
-                    - name: FF_MAX_FOLLOWERS
-                      value: "10"
-
-                    # Add any other options below as described in in the README.md file
-
-                    # If you don't want to use a PVC you may comment the next two lines, but that will significantly 
-                    # affect performance, and is NOT recommended
-                    - name: FF_STATE_DIR
-                      value: "/data/"
-                    - name: FF_SERVER
-                      valueFrom: 
-                        secretKeyRef:
-                            name: fedifetcher
-                            key: server_domain
-                            optional: false
-                    - name: FF_ACCESS_TOKEN
-                      valueFrom: 
-                         secretKeyRef:
-                            name: fedifetcher
-                            key: token
-                            optional: false
-                # Comment the lines below if you do not use a PVC, but that will significantly 
-                # affect performance and is NOT recommended
-                volumeMounts:
-                    - mountPath: /data
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: fedifetcher
+              image: ghcr.io/nanos/fedifetcher:latest
+              imagePullPolicy: IfNotPresent
+              env:
+                - name: FF_HOME_TIMELINE_LENGTH
+                  value: "200"
+                - name: FF_MAX_FOLLOWERS
+                  value: "10"
+                # Add any other options below as described in in the README.md file
+    
+                # If you don't want to use a PVC you may comment the next two lines, but that will significantly 
+                # affect performance, and is NOT recommended
+                - name: FF_STATE_DIR
+                  value: "/data/"
+                - name: FF_SERVER
+                  valueFrom:
+                    secretKeyRef:
+                      name: fedifetcher
+                      key: server_domain
+                - name: FF_ACCESS_TOKEN
+                  valueFrom:
+                    secretKeyRef:
                       name: fedifetcher
-                      readOnly: false 
```

Comment:
why is this no longer read-only?

Reply:
You are correct here, added again. 

I had two versions of the file, as I did things to the version I use for my cluster that might be confusing in an example as it very specific for the cluster I use.